Makefile

```diff
@@ -205,20 +205,6 @@ clean: $(foreach SDIR, $(CLEANDIRS), $(SDIR)_clean)
 	$(call CLEAN)
 
 distclean: $(foreach SDIR, $(CLEANDIRS), $(SDIR)_distclean)
-ifeq ($(CONFIG_WINDOWS_NATIVE),y)
```

Comment:
But does testbuild.sh suitable for windows native build?

Reply:
Linux/macOS CI could catch all path since they cover all defined config except a few ones.